scripts/ci/run-unittest.py

```
@@ -102,11 +128,19 @@ def main(argv: list[str] | None = None) -> int:
     parser.add_argument("--profile", default="all")
-    parser.add_argument("--size", default="small", help="размер набора для метки")
+    parser.add_argument("--size", default="small", help="размер набора: умолчание для тестов без записи в манифесте")
+    parser.add_argument("--sizes", type=Path, default=None, help="манифест размеров .config/python-sizes.toml")
+    parser.add_argument("--admit", default="", help="какие размеры гонять, через запятую; пусто — все")
     parser.add_argument("--plan-only", action="store_true", help="перечислить тесты и выйти")
     args = parser.parse_args(argv)
 
+    sizes = Sizes.load(args.sizes, args.size)
+    admitted = {size.strip() for size in args.admit.split(",") if size.strip()}
     suite = unittest.defaultTestLoader.discover(args.start_directory)
+    # Не допущенный воротами тест — не в этом плане: его результат в отчёте
```

Ruff (0.16.3)

[warning] Comment contains ambiguous `Н` (CYRILLIC CAPITAL LETTER EN). Did you mean `H` (LATIN CAPITAL LETTER H)?

(RUF003)